go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/rclone/rclone
 
-go 1.23.0
+go 1.24.0
```

Comment:
This should not be changed. Rclone's policy is to support 2 versions of Go like most of the Go ecosystem.